packages/riipen-ui/src/components/MenuList.test.jsx

```diff
@@ -0,0 +1,299 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import MenuItem from "./MenuItem";
+import MenuList from "./MenuList";
+
+describe("<MenuList>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(
+        <MenuList>
+          <MenuItem />
+        </MenuList>
+      );
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(
+      <MenuList>
+        <MenuItem />
+      </MenuList>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new MenuList().type.defaultProps;
+
+      const wrapper = mount(
+        <MenuList>
+          <MenuItem />
+        </MenuList>
+      );
+
+      const component = wrapper.find("MenuList");
+      expect(component.props().selectedIndex).toEqual(
+        defaultProps.selectedIndex
+      );
+      expect(component.props().variant).toEqual(defaultProps.variant);
+    });
+  });
+
+  describe("autoFocus prop", () => {
+    it("selects first element on list open with true autoFocus and default selectedIndex", () => {
```

Comment:
This sentence is sort of confusing to me. I think what we are testing is that we auto focus the first element of the list on mount, so what if we said "selects first item of list with ..."